[2024-05-14T05:27:19.895+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T05:27:19.928+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tugas_workflow_v1.task_6 scheduled__2023-11-08T00:00:00+00:00 [queued]>
[2024-05-14T05:27:19.942+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tugas_workflow_v1.task_6 scheduled__2023-11-08T00:00:00+00:00 [queued]>
[2024-05-14T05:27:19.943+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-14T05:27:19.969+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_6> on 2023-11-08 00:00:00+00:00
[2024-05-14T05:27:19.980+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2881) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T05:27:19.983+0000] {standard_task_runner.py:63} INFO - Started process 2923 to run task
[2024-05-14T05:27:19.993+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tugas_workflow_v1', 'task_6', 'scheduled__2023-11-08T00:00:00+00:00', '--job-id', '517', '--raw', '--subdir', 'DAGS_FOLDER/__pycache__/prioritas_1.py', '--cfg-path', '/tmp/tmpsgr6hhvl']
[2024-05-14T05:27:20.007+0000] {standard_task_runner.py:91} INFO - Job 517: Subtask task_6
[2024-05-14T05:27:20.106+0000] {task_command.py:426} INFO - Running <TaskInstance: tugas_workflow_v1.task_6 scheduled__2023-11-08T00:00:00+00:00 [running]> on host 0391a550332e
[2024-05-14T05:27:20.335+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sinta' AIRFLOW_CTX_DAG_ID='tugas_workflow_v1' AIRFLOW_CTX_TASK_ID='task_6' AIRFLOW_CTX_EXECUTION_DATE='2023-11-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-08T00:00:00+00:00'
[2024-05-14T05:27:20.341+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T05:27:20.343+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-14T05:27:20.345+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "done!"']
[2024-05-14T05:27:20.377+0000] {subprocess.py:86} INFO - Output:
[2024-05-14T05:27:20.380+0000] {subprocess.py:93} INFO - done!
[2024-05-14T05:27:20.381+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-14T05:27:20.382+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T05:27:20.472+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=tugas_workflow_v1, task_id=task_6, run_id=scheduled__2023-11-08T00:00:00+00:00, execution_date=20231108T000000, start_date=20240514T052719, end_date=20240514T052720
[2024-05-14T05:27:20.607+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T05:27:20.641+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-14T05:27:20.656+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
